utils/avx512_calls.h

```diff
@@ -282,7 +283,7 @@ extern "C" {
 #define AVX512_RECIPROCAL_FLOATS(X)                               _mm512_rcp14_pd(X)
 #define AVX512_MASK_RECIPROCAL_FLOATS(FALSEVALS, MASK, X)         _mm512_mask_rcp14_pd(FALSEVALS, MASK, X)
 #define AVX512_MASKZ_RECIPROCAL_FLOATS(MASK, X)                   _mm512_maskz_rcp14_pd(MASK, X)
-
+#define AVX512_CONVERT_INT_TO_FLOAT(X)                            _mm512_cvtepi32_pd(X)
```

Comment:
I am wondering if you need the (instruction-less) cast to float ([these ops](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#avx512techs=AVX512F&text=_mm512_castsi512_p&expand=723)) rather than the expensive convert options